van-gogh-second-page/maria/index-Van-Gogh-for-Children.css

```diff
@@ -0,0 +1,220 @@
+
+* {
+    margin: 0px;
+    padding: 0px;
+}
+
+input, button, submit { border:none; background: none; } 
+
+a {
+    background-color: none;
+    text-decoration: none;
+    color: inherit;
+}
+
+/* Styles */
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    background-color: #FBAE17;
+   
+}
+.everything-container
+{
+    overflow-y: auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    max-width: 2000px;
+    
+
+}
+
+header {
+    padding-left: 5%;
+    padding-right: 5%;
+    box-sizing: border-box;      /* prevent padding from adding extra width */
+    width: 100%;
+    padding-top: 40px;
+    height: 80px;
+    display: flex;
+    /* position: fixed; */
+    /* justify-content: flex-start; */
+    justify-content: space-between;
+    align-items: flex-start;
+    /* background-color:#DACC98; */
+}
+header nav {
+    display: flex; 
+}
+
+.title
+{
+    padding-left: 5%;
+    padding-right: 5%;
+    padding-top: 5%;
+}
+h1
+{
+    font-size: 80px;
+    font-weight: 500;
+}
+.title-description
+{
+    padding-top: 3%;
+}
+.navigation
+{
+    display: flex;
+    padding-left: 5%;
+    padding-right: 10%;
+    padding-top: 5%;
+    justify-content: space-between;
+}
+.navigation-item
+{
+    font-weight: 550;
```

Comment:
I don't think we have this value, some command values: 
200 - light
400 - normat
500 - medium
700 - bolt